Dockerfile.mri.erb

```diff
@@ -58,14 +58,16 @@ RUN bash -c " \
       for v in ${RVM_RUBIES} ; do \
         rvm install \$v --patch \$(echo ~/patches/ruby-\$v/* | tr ' ' ','); \
       done && \
-      rvm cleanup all \
+      rvm cleanup all && \
+      find /usr/local/rvm -type d -print0 | sudo xargs -0 chmod g+sw \
     "
 
 RUN bash -c " \
       echo "gem: --no-ri --no-rdoc" >> ~/.gemrc && \
       rvm all do gem update --system --no-document && \
       rvm all do gem update --no-document && \
-      rvm all do gem install bundler --no-document \
+      rvm all do gem install bundler --no-document && \
+      find /usr/local/rvm -type d -print0 | sudo xargs -0 chmod g+sw \
```

Comment:
I think "gem inst" doesn't break the group bits, but better once more than too few.